wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/Button.java

```diff
@@ -91,7 +104,9 @@ public Button whileHeld(final Runnable toRun, Subsystem... requirements) {
    *
    * @param command the command to start
    * @return this button, so calls can be chained
+   * @deprecated Replace with {@link Trigger#whileTrue(Command)}
```

Comment:
Is this an exact replacement? both whileHeld and whenHeld recommend the same replacement which doesn't seem correct.

Reply:
Now pushed a fix for Java.
C++ deprecation messages were already correct.

Both `whileHeld` and `whenHeld` should still be replaced with `whileTrue`, but for `whileHeld`-esque behavior the command needs to be put in a `RepeatCommand` (or replaced by a `RunCommand` if it's an `InstantCommand`).